### ISSUE TITLE
CLI: `fluvio version` should report versions of all extensions

### ISSUE THREAD

**Issue by nicholastmosher** [infinyon/fluvio#673]
Right now, `fluvio version` only shows the current version of the top-level Fluvio CLI itself. It should also show the versions of any installed extensions, such as `fluvio cloud`. We can potentially improve our extension-metadata system to help us implement this.

Example desired output:

```
$ fluvio version
Fluvio CLI      : 0.6.0-rc.5
Fluvio Platform : 0.6.0-rc.5
Fluvio Cloud    : 0.1.1
Other extension : x.y.z
Git Commit      : 93193ab9754fb0019984c704640087ed7ba1420b
OS Details      : Darwin 19.6.0 x86_64
Rustc Version   : 1.48.0 (7eac88a 2020-11-16)
```

**Comment by ajhunyady** [comment]
Suggestion: Version command should only display installed extensions (ignore others)

**Comment by nicholastmosher** [comment]
Yes, certainly. We don't want to list off the versions of all available extensions (e.g. on the registry), only the versions of installed items 👍

**Comment by sehz** [comment]
We should make it version uniform, here's suggested output:
```
$ fluvio version
Fluvio CLI      : 0.6.0-rc.5
Fluvio Platform : 0.6.0-rc.5
Git Commit      : 93193ab9754fb0019984c704640087ed7ba1420b
OS Details      : Darwin 19.6.0 x86_64

Extensions
Fluvio Cloud
     Version: 0.1.1
     Git Commit: AAAAAAAAAA
Other extension : x.y.z
    Git Commit:  BBBBBBBB


```

**Comment by nicholastmosher** [comment]
Closed by #854
